std_msgs/Header header
int32 hand_id
int32 finger_id
int32 id
float32 time_visible
geometry_msgs/Vector3 direction
float32 angle
